modules/vcf2maf/1.3/vcf2maf.smk

```diff
@@ -166,6 +185,140 @@ rule _vcf2maf_gnomad_filter_maf:
         touch {output.dropped_maf}
         """)
 
+# Obtain the path to the GAMBLR conda environment
+md5hash = hashlib.md5()
+if workflow.conda_prefix: 
+    conda_prefix = workflow.conda_prefix
+else: 
+    conda_prefix = os.path.abspath(".snakemake/conda")
+
+md5hash.update(conda_prefix.encode())
+f = open("config/envs/GAMBLR.yaml", 'rb')
+md5hash.update(f.read())
+f.close()
+h = md5hash.hexdigest()
+GAMBLR = glob.glob(conda_prefix + "/" + h[:8] + "*")[0]
+
+rule _vcf2maf_install_GAMBLR:
```

Comment:
I think this is missing from the local rules directive above, just want to confirm if this is intended

Reply:
Good catch, should definitely be a localrule